addons/sale_management/models/sale_order.py

```diff
@@ -4,7 +4,8 @@
 from datetime import datetime, timedelta
 
 from odoo import api, fields, models, _
-from odoo.exceptions import UserError
+from odoo.addons import decimal_precision as dp
```

Comment:
bad import, probably a rebase issue (decimal_precision is part of base now, no import needed)

Reply:
done.